src/preferences/dialog/dlgprefdeck.h

```diff
@@ -22,6 +22,10 @@ namespace TrackTime {
         Remaining,
         ElapsedAndRemaining,
     };
+    enum class DisplayFormat {
+        DEFAULT,
```

Comment:
DEFAULT is meaningless here. How about WALL_CLOCK or HOUR_MINUTES_SECONDS or TRADITIONAL